includes/class-used-css.php

```
@@ -0,0 +1,1206 @@
+					$local_path = Util::get_local_path( $src );
+					if ( ! empty( $local_path ) ) {
+						$min_file = wp_normalize_path( WP_CONTENT_DIR . '/cache/wppo/min/css/' . basename( $local_path ) );
+						if ( file_exists( $min_file ) ) {
+							$removal_urls[ content_url( 'cache/wppo/min/css/' . basename( $min_file ) ) ] = true;
+						}
+					}
+				}
+			}
+
+			// Remove original <link> tags.
+			foreach ( $removal_urls as $url => $true ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$true'. (undefined)

(UnusedLocalVariable)